bsdlib/README.rst

```diff
@@ -3,9 +3,131 @@
 BSD Library
 ###########
 
+The BSD library is Nordic Semiconductor's implementation of the BSD Socket API, which is a set of standard function calls that can be used in an application.
```

Comment:
Can we add a link to BSD Socket API?
(For example, https://en.wikipedia.org/wiki/Berkeley_sockets#Socket_API_functions )

Reply:
Oh, and do we know what BSD actually stands for? In the picture it says Berkeley Socket Descriptors, but Google has never heard of this term ...

Reply:
BSD seems to stand for just Berkely Sockets in this case, or that seems be what everyone refer to, so I am not sure where the D is lost.

Reply:
Actually BSD stands for Berkeley Software Distribution, so it's BSD Sockets is a part of Berkeley Software Distribution interfaces.

Reply:
That's interesting. I'll just change to BSD in the picture to be on the safe side and follow up with Glenn later.